### PR TITLE
feat: new website

### PR DESCRIPTION
**Pull request by lucaburgio**
I did a restyle and some other changes to the website. Take your time, and let me know if this looks good!

### CODE REVIEW

**Review by sammarks — APPROVED** [review]
I don't have the time for a full detailed review currently regarding code style, best practices, etc, but I trust you've done detailed testing locally and it's all working OK.

LGTM 👍🏼

**Comment by paescuj** [comment]
Looks like you've done some great work here! 🚀 I can take a closer look at it tonight if you can wait until then 😄

**Comment by lucaburgio** [comment]
@paescuj Let me know if you prefer to skip this. No problem at all if you have no time right now 🙌

File: iconoir.com/components/Explore.tsx [hunk 1]
```diff
@@ -49,21 +49,29 @@ export function Explore({ allIcons }: ExploreProps) {
...
+  flex-direction: column-reverse;
+  ${media.md} {
+    align-items: flex-start;
+    flex-direction: row;
+  }
 `;
 const Left = styled.div`
   flex: 1;
   min-height: calc(100vh - 100px);
+  background:white;
```
Comment (paescuj) [inline iconoir.com/components/Explore.tsx:61]:
```suggestion
  background: white;
```

File: iconoir.com/components/HeaderBackground.tsx [hunk 2]
```diff
@@ -1,46 +1,124 @@
...
 import { media } from '../lib/responsive';
+import {useRef, useEffect} from 'react';
 
 export interface HeaderBackgroundProps {
   children: React.ReactElement;
-  src: string;
 }
-export function HeaderBackground({ children, src }: HeaderBackgroundProps) {
+export function HeaderBackground({ children }: HeaderBackgroundProps) {
+
```
Comment (paescuj) [inline iconoir.com/components/HeaderBackground.tsx:10]:
```suggestion
```

File: iconoir.com/components/HeaderBackground.tsx [hunk 3]
```diff
@@ -1,46 +1,124 @@
...
       {children}
     </HeaderContainer>
   );
 }
 
 const HeaderContainer = styled.div`
   position: relative;
+  width:fit-content;
+  margin:auto;
+  /*background-color:green;*/
```
Comment (paescuj) [inline iconoir.com/components/HeaderBackground.tsx:51]:
```suggestion
  width: fit-content;
  margin: auto;
```

File: iconoir.com/components/HeaderSecondary.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      {children}
+    </HeaderContainer>
+  );
+}
+
+const HeaderContainer = styled.div`
+  position: relative;
+  width:fit-content;
+  margin:auto;
+  /*background-color:green;*/
```
Comment (paescuj) [inline iconoir.com/components/HeaderSecondary.tsx:22]:
```suggestion
  width: fit-content;
  margin: auto;
```

File: iconoir.com/components/Input.tsx [hunk 5]
```diff
@@ -50,8 +50,10 @@ export const LargeInput = styled(Input)`
   width: 100%;
   box-sizing: border-box;
   outline: none;
+  transition: 0.2s;
   &:hover{
-    transition: 0.4s;
+    transition: 0.2s;
```
Comment (paescuj) [inline iconoir.com/components/Input.tsx:55]: Already on root
```suggestion
```

File: iconoir.com/pages/index.tsx [hunk 6]
```diff
@@ -81,38 +82,48 @@ const Home: NextPage<HomeProps> = ({
...
+    margin: 160px auto 80px auto;
+  }
+`;
 export const HeroText = styled.h1`
   font-size: 50px;
   font-weight: 700;
-  letter-spacing: -0.05em;
-  line-height: 52px;
+  color: var(--g0);
+  margin:0 auto;
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:107]:
```suggestion
  margin: 0 auto;
```

File: iconoir.com/pages/index.tsx [hunk 7]
```diff
@@ -81,38 +82,48 @@ const Home: NextPage<HomeProps> = ({
...
+  letter-spacing: -0.035em;
+  line-height: 1;
   text-align: center;
-  margin: 60px auto 40px auto;
-
+  transition:0.3s;
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:111]:
```suggestion
  transition: 0.3s;
```

File: iconoir.com/pages/index.tsx [hunk 8]
```diff
@@ -81,38 +82,48 @@ const Home: NextPage<HomeProps> = ({
...
   ${media.md} {
     font-size: 90px;
-    line-height: 1;
-    margin: 200px auto 80px auto;
+    padding:0 20px;
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:114]:
```suggestion
    padding: 0 20px;
```

File: iconoir.com/pages/index.tsx [hunk 9]
```diff
@@ -81,38 +82,48 @@ const Home: NextPage<HomeProps> = ({
...
+    letter-spacing: -0.045em;
+    -webkit-text-stroke: 2.5px;
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:116]:
```suggestion
    -webkit-text-stroke: 2.5px;
    text-stroke: 2.5px;
```

File: iconoir.com/pages/index.tsx [hunk 10]
```diff
@@ -81,38 +82,48 @@ const Home: NextPage<HomeProps> = ({
...
   }
+  &:hover {
+    transform:scale(1.04);
+    transition:0.3s;
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:120]:
```suggestion
    transform: scale(1.04);
```

File: iconoir.com/pages/index.tsx [hunk 11]
```diff
@@ -81,38 +82,48 @@ const Home: NextPage<HomeProps> = ({
...
+  }
+  
+`;
+export const HeroTextSecondary = styled(HeroText)`
+color: var(--g4);
+max-width: 1000px;
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:126]:
```suggestion
  color: var(--g4);
  max-width: 1000px;
```

File: iconoir.com/pages/index.tsx [hunk 12]
```diff
@@ -171,4 +176,10 @@ export async function getStaticProps() {
       numDownloads,
     },
   };
+
+  
+
+
+
```
Comment (paescuj) [inline iconoir.com/pages/index.tsx:183]:
```suggestion
```

File: iconoir.com/pages/support.tsx [hunk 13]
```diff
@@ -98,25 +82,97 @@ const Support: NextPage<SupportProps> = ({ ...headerProps }) => {
...
+  padding: 60px 10%;
+  margin-bottom: 20px;
+  ${media.md} {
+    width: 22%;
+    height: 180px;
+    padding: 60px 5%;
+    margin-bottom: 0;
+    &:hover {
+      transform: scale(1.05);
+      transition: 0.2s
```
Comment (paescuj) [inline iconoir.com/pages/support.tsx:116]: Moving to root so it takes effect on mouse out as well
```suggestion
    transition: 0.2s;
    &:hover {
      transform: scale(1.05);
```

File: iconoir.com/pages/support.tsx [hunk 14]
```diff
@@ -98,25 +82,97 @@ const Support: NextPage<SupportProps> = ({ ...headerProps }) => {
   );
 };
 
+const CenteredParagraph = styled(Text18)`
+  text-align: center;
+  margin: 120px auto;
+`;
```
Comment (paescuj) [inline iconoir.com/pages/support.tsx:88]: Not used
```suggestion
```

File: iconoir.com/pages/support.tsx [hunk 15]
```diff
@@ -98,25 +82,97 @@ const Support: NextPage<SupportProps> = ({ ...headerProps }) => {
...
+const PillarIcon = styled.span<{ R?: string, G?: string, B?: string  }>`
+  font-size: 18px;
+  padding: 20px 11px 6px 11px;
+  color: var(--white);
+  width: 100px;
+  height: 100px;
+  background: rgb(${({ R }) => R}, ${({ G }) => G}, ${({ B }) => B});
+  box-shadow: 2px 6px 20px rgba(${({ R }) => R}, ${({ G }) => G}, ${({ B }) => B}, 0.5);
+  border-radius: 18px;
+
```
Comment (paescuj) [inline iconoir.com/pages/support.tsx:141]:
```suggestion
```

File: iconoir.com/components/Button.tsx [hunk 16]
```diff
@@ -64,3 +68,21 @@ export const Button = styled(LargeButton)`
...
+  letter-spacing: 0.12em;
+  line-height: 17.6px;
+  font-weight: 700;
+  color: var(--black);
+  transition: background 0.1s linear, color 0.1s linear;
+  font-family: var(--font-family) !important;
+  &:hover {
+    background: var(--g0) !important;
+    color: var(--white);
+  }
```
Comment (paescuj) [inline iconoir.com/components/Button.tsx:87]: Do we really want this color transformation? The other buttons don't have it.

### Before

https://user-images.githubusercontent.com/5363448/226179145-e985a76a-972e-4284-9a05-a87a07e657e6.mp4

### After

https://user-images.githubusercontent.com/5363448/226179161-466b31e9-abb5-4ac2-862c-b79c70c5fc75.mp4

```suggestion
  font-family: var(--font-family) !important;
```

**Review by paescuj — COMMENTED** [review]
Looks great on the whole! 🚀  I have only a few minor comments.

Also noticed that the ad seems to be displayed incorrectly on the smartphone:

https://user-images.githubusercontent.com/5363448/226179286-e763023e-5e7c-4724-a092-50b8535814e9.mp4

File: iconoir.com/pages/support.tsx (hunk 14 above)
Comment (lucaburgio) [reply iconoir.com/pages/support.tsx:88]: Thank you for spotting this 🙏

File: iconoir.com/components/Icon.tsx [hunk 17]
```diff
@@ -111,6 +115,31 @@ export function Icon({ iconWidth, icon }: IconProps) {
...
+right: -6px;
+`;
+const CornerBL = styled(Test)`
+bottom: -6px;
+left: -6px;
+`;
+const CornerTL = styled(Test)`
+top: -6px;
+left: -6px;
+`;
```
[the remark this answers is not in the document]
Comment (lucaburgio) [reply iconoir.com/components/Icon.tsx:142]: Oh, thank you for spotting it! I totally forgot to rename it.

File: iconoir.com/components/Button.tsx (hunk 16 above)
Comment (lucaburgio) [reply iconoir.com/components/Button.tsx:85]: Yeah, I agree. I wasn't against the effect while working on it, but it absolutely makes sense to merge it with all the other buttons 🙌

File: iconoir.com/components/HeaderBackground.tsx [hunk 18]
```diff
@@ -1,46 +1,124 @@
...
+  -webkit-transform: rotate(-14deg);
+    -moz-transform: rotate(-14deg);
+  top: -64px;
+  left: -75px;
+  width:110px;
+  height:110px;
+  background-image: url(/fill.gif);
+  background-size:70%;
+  ${media.lg} {
   }
```
[the remark this answers is not in the document]
Comment (lucaburgio) [reply iconoir.com/components/HeaderBackground.tsx:121]: I actually wanted to add different rotations to each icon. So I would keep it for now, but I saved this and I will apply all the other changes you suggested

File: iconoir.com/pages/support.tsx (hunk 13 above)
Comment (lucaburgio) [reply iconoir.com/pages/support.tsx:112]: I intentionally wanted to have a faster transition on `mouse out` to have a quicker interaction with the button. But GTM, let's see how this looks, no strong point 🙌

File: iconoir.com/pages/index.tsx [hunk 19]
```diff
@@ -36,8 +34,11 @@ const Home: NextPage<HomeProps> = ({
     <Layout>
       <SEO />
       <Header currentVersion={currentVersion} />
-      <HeaderBackground src={'/home-background.svg'}>
-        <HeroText>Your new default library.</HeroText>
+      <HeaderBackground>
+        <HeroHead>
+        <HeroText>Say hello</HeroText>
+        <HeroTextSecondary>to your new icons library.</HeroTextSecondary>
```
[the remark this answers is not in the document]
Comment (lucaburgio) [reply iconoir.com/pages/index.tsx:40]: Makes totally sense! 👍

**Comment by lucaburgio** [comment]
Thank you for all the suggestions! I commented and merged them 🙌
I'll fix the ad on mobile later. Duly noted. Thank you for spotting it!

**Comment by paescuj** [comment]
> Thank you for all the suggestions! I commented and merged them 🙌 I'll fix the ad on mobile later. Duly noted. Thank you for spotting it!

You're very welcome! Awesome, looks great 😃❤️

**Pull request merged**